guides/concepts/list-of-operations.md

```diff
@@ -413,9 +429,12 @@ Possible errors:
 ## Bump Sequence
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.bumpSequence) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/BumpSequenceOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#BumpSequence)
 
-*Only available in protocol version 10 and above*
+*Only available in Stellar Protocol >= v10*
+
+Bump forward the sequence number of the source account to the given sequence number.
 
-Bump sequence allows to bump forward the sequence number of the source account of the operation, allowing to invalidate any transactions with a smaller sequence number.
+This operation allows you to invalidate any transactions with a smaller sequence number, and is
```

Comment:
`This operation allows you to invalidate any` -> `This operation invalidates any`